config/crd/bases/infrastructure.cluster.x-k8s.io_azureclustertemplates.yaml

```diff
@@ -432,6 +432,26 @@ spec:
                         - name
                         - type
                         type: object
+                      failureDomains:
```

Comment:
idk if this should go into `azureclustertemplates `as well or just `azureclusters`